blosc/dframe.c

```diff
@@ -56,7 +56,7 @@ void* eframe_create_chunk(blosc2_frame* frame, uint8_t* chunk, int32_t nchunk, i
 
 
 /*Get chunk from extended frame. */

```

Comment:
Ditto.

Perhaps you can do a (supervised) automatic replace for extended - > disperse in the code too.

Reply:
Fix in 2426e5ae2a82ab94f3fc05902aeb22df78beb0b3 .